models/intel/action-recognition-0001/description/action-recognition-0001.md

```diff
@@ -0,0 +1,68 @@
+# Composite model name: action-recognition-0001
+
+## Use Case and High-Level Description
+
+This is a general-purpose action recognition composite model, consisting from encoder and decoder parts, trained on Kinetics-400 dataset. The encoder model uses Video Transformer approach with ResNet34 encoder.
+Please refer to the [kinetics](https://deepmind.com/research/open-source/open-source-datasets/kinetics/) dataset specification to see list of action that are recognised by this composite model.
+
+## Example
+
+![](./demo.png)
+
+## Encoder model specification
+
+The [encoder model](../action-recognition-0001-encoder) accepts video frame and produces embedding.
+Video frames should be sampled to cover ~1 second fragment (i.e. skip every second frame in 30 fps video).
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Source framework                | PyTorch*                                  |
+| GFlops                          | 7.340                                     |
+| MParams                         | 21.276                                    |
+
+
+## Performance
```

Comment:
```suggestion
### Performance
```

These are per-model headings, so they should be level 3.

Reply:
changed